components/settings/changePasswordStepper/PswrdChangeStepSecond.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, Button, Typography, alpha, useTheme } from "@mui/material";
```

Comment:
Could you make dummy POST request which will respond with Promise object with a operation status? like status: 'failed' | 'success' | 'wrong-code'? Then if the code was wrong "NASTĘPNY" button will be blocked and maybe some error color text will be displayed under "WYŚLIJ PONOWNIE" e.g. "Podany kod jest błędny". If its failure (means something wrong with backend) button "NASTĘPNY" won't be disabled but final step will be as error in figma. And last success as it is now

Reply:
fixed